itools/database/registry.py

```diff
@@ -38,12 +37,15 @@ def register_field(name, field_cls):
         old_value = getattr(old, key, None)
         new_value = getattr(new, key, None)
         if type(old_value) is MethodType:
-            old_value = old_value.im_func
+            old_value = old_value.__func__
         if type(new_value) is MethodType:
-            new_value = new_value.im_func
+            new_value = new_value.__func__
+
         if old_value != new_value:
             msg = 'register conflict over the "{0}" field ({1} is different)'
-            raise ValueError(msg.format(name, key))
+            pass
+            #### Python 3 pass error
+            #raise ValueError(msg.format(name, key))
```

Comment:
raise à remettre ici ?

Reply:
Erreurs rencontrées avec ikaaro. Nous n'avons pas trouvé de solution avec Juan David

Reply:
C'est à dire ? Cette vérification et exception sert à éviter des conflits d'attributs de fields, un field du même nom déclaré avec indexed=True et un autre du même nom avec indexed=False

Reply:
Traceback (most recent call last):
  File "/home/elie/anaconda3/bin/icms-start.py", line 72, in <module>
    server = Server(target, read_only=options.read_only,
  File "/home/elie/anaconda3/lib/python3.8/site-packages/ikaaro/server.py", line 403, in __init__
    load_modules(config)
  File "/home/elie/anaconda3/lib/python3.8/site-packages/ikaaro/server.py", line 182, in load_modules
    __import__(name)
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/__init__.py", line 51, in <module>
    from .root import BP_Root as Root
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/root.py", line 47, in <module>
    from bepatient.context import BepatientCMSContext
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/context.py", line 32, in <module>
    from .websites import Website_Base
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/websites/__init__.py", line 6, in <module>
    from .websites import Websites_Field, AvailableWebsites_Field, Website_Base
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/websites/websites.py", line 43, in <module>
    from .website_menu import Bepatient_ConfigMenu
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/websites/website_menu.py", line 20, in <module>
    from bepatient.api.utils import get_item_link
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/api/utils.py", line 22, in <module>
    from bepatient.users.api import ProfileFormAnswerUUIDView
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/users/api.py", line 31, in <module>
    from bepatient.mail.api import DiscussionUUIDView, DiscussionsView
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/mail/api.py", line 22, in <module>
    from .resources import MessageType_Field
  File "/home/elie/anaconda3/lib/python3.8/site-packages/bepatient/mail/resources.py", line 321, in <module>
    class Mail_Discussion(BePatientFolder):
  File "/home/elie/anaconda3/lib/python3.8/site-packages/itools-python3.dev202204151622-py3.8-linux-x86_64.egg/itools/database/resources.py", line 47, in __new__
    register_field(name, datatype)
  File "/home/elie/anaconda3/lib/python3.8/site-packages/itools-python3.dev202204151622-py3.8-linux-x86_64.egg/itools/database/registry.py", line 46, in register_field
    msg = 'register conflict over the "{0}" field ({1} is different)'
ValueError: register conflict over the "cc_list" field (default is different)